app/src/main/java/ar/com/wolox/android/example/ui/example/ExampleFragment.kt

```diff
@@ -23,8 +24,13 @@ class ExampleFragment private constructor() : WolmoFragment<FragmentExampleBindi
         }
     }
 
-    override fun toggleLoginButtonEnable(isEnable: Boolean) {
-        binding.loginButton.isEnabled = isEnable
+    override fun checkErrors(Errors: MutableMap<String, MutableList<String>>) {
+        Errors.forEach { (s, mutableList) ->
+            if (mutableList.isNotEmpty()) {
+                if (s == "email") binding.email.error = mutableList.joinToString { "$it \n" }
+                if (s == "password") binding.password.error = mutableList.joinToString { "$it \n" }
```

Comment:
There are 2 things to talk about here.
First, this is ok, because you only have 2 editText, but it isn't really good if you have more and more editText. Maybe you can find a way to avoid checking the error by a key. 
Second, in both cases you do a `mutableList.joinToString { "$it \n" }`. So, does it matters the map key if you apply the same action?